app/App.hx

```diff
@@ -44,14 +44,18 @@ class App
         if (!relay && FileSystem.exists("combo.txt"))
         {
             var lines = File.getContent("combo.txt").split("\r\n");
-            Sys.println("(y)es (n)o deploy " + lines.length + " bots:");
+            var botamount = lines.length;
```

Comment:
So like here I made it adapt to the amount of bots in the combo.txt file because mine had less than 20 :P